### DISCUSSION TITLE
I can't get events from Wazuh-agent on Centos

### DISCUSSION THREAD

**Discussion by najiyousif** [Security-Onion-Solutions/securityonion#5865] [Unsupported Versions]
Hi Guys,

I'm newbie.
I have just configured SO,and deployed wazuh agent, everything is fine with Windows machines, but can't receive anything from CentOS,

so-wazuh-agent-manage displaying CentOS machines, 
Firewall-cmd allowing ports 1515,1514
I've extracted agent key & added it using ./manage_agents

any ideas??

**Comment by dougburks** [comment]
Are there any network firewalls separating your CentOS machines from your Security Onion box that could be interfering with the traffic?

Where are you looking for events?
